### PR TITLE
Rs 2022 07 23.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-07-23-221152_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 1]
```diff
@@ -177,87 +192,101 @@ It's not like I hate you or anything."
...
 -- 「……いいです、言い訳なんかしなくても。
-"You don't have to make any excuses.
+"That's okay... you don't have to make excuses.
 }
 [sha:e04f78ebd87653b0221334561c942e3b12964b0c]{
 -- Page 17, Offset 8231.
 -- わたし、やっぱりお節介でしたよね……あと夕ご飯、コンビニ弁当は女子力低すぎでしたよね……」
-I knew I was meddling... Also, serving convenience store bentos for dinner isn't very feminine..."
+I overstepped my bounds, didn't I...
+ Plus, serving convenience store food for dinner doesn't reflect well on me as a girl..."
```
Comment (yk1512) [inline script/Ciel/Day 4/04_01_CIEL04_1.txt:201]: japanese convenience store food is romantic af dw ciel

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 2]
```diff
@@ -56,7 +61,8 @@ Thanks a lot for yesterday. You did a lot for me――――"
 [sha:bf1957b3a56494f464ee415fea7aa98a600a6290]{
 -- Page 5, Offset 8206.
 -- 慌てている俺と違って、先輩はいつもの笑顔だ。
-Unlike me, who was in a panic, Senpai had her usual smile.
+I'm a flustered mess,
+ but Senpai has the same smile on her face she always does.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:65]: I disagree, I think "she always does" sounds normal

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 3]
```diff
@@ -67,107 +73,116 @@ She really is in another league.
...
 -- Page 8, Offset 8213.
 -- 「あれ、そうなんですか？
-"Oh, is that so?
+"Huh, are you sure?
 }
 [sha:884e4b6c96dbef97938e778e38c81381f64031d0]{
 -- Page 8, Offset 8214.
 -- まだ朝の６時になったばかりですよ、遠野くん」
-It's only six in the morning, Tohno-kun."
+It's only just gone six o'clock you know, Tohno-kun."
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:109]: indeed it is a gb thing. "It's only six o'clock you know, Tohno-kun" would be the most natural. But if the になったばかり is of the utmost importance, "It's only just now six o'clock" would be the american version

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 4]
```diff
@@ -67,107 +73,116 @@ She really is in another league.
...
 -- わなわなと震える喉。
-My throat quivered in fear.
+My voice is trembling with fear.
 }
 [sha:2fa4ca37b7c25482dbaf293a28079d556cf2abf7]{
 -- Page 13, Offset 8225.
 -- シエル先輩がうちに電話したっていう事は、つまり、同年代の女生徒が『遠野志貴はうちに泊まります』と、あのお厳しい当主様に伝えたという事だ。
-Then that means, a young woman; one around my age, told our strict family head I'd be staying at her place.
+Ciel-senpai called the mansion.
+ In other words, a female student the same age as me called up our totalitarian family head and told her that
```
Comment (Avuxo) [inline script/Ciel/Day 4/04_01_CIEL04_1.txt:166]: > told her: 'Tohno

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 5]
```diff
@@ -67,107 +73,116 @@ She really is in another league.
...
+ 'Tohno Shiki is spending the night with me'.
 }
 [sha:2f148ee523e3f9335a9002c9d42a0ac0b318fa3d]{
 -- Page 14, Offset 8226.
 -- 「―――困る。それは、すごく困る。昨日とは違う意味で、死んだ方がマシかもしれない」
-"This is bad, very bad. In a different way than yesterday, I might as well be dead."
+"This is bad. This is %{i}really%{/i} bad.
+ I'd probably be better off dead, though for a different reason than I thought so yesterday."
```
Comment (Avuxo) [inline script/Ciel/Day 4/04_01_CIEL04_1.txt:173]: than I thought yesterday

File: script/Ciel/Day 4/04_01_CIEL04_1.txt (hunk 1 above)
Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:201]: image

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 6]
```diff
@@ -177,87 +192,101 @@ It's not like I hate you or anything."
...
-"I see. You're back to normal. I don't know what happened last night, but I'm glad you're feeling better."
+"Ah, there we go.
+ Seems like you're all back to normal.
+ I don't know what happened last night, but I'm glad you're feeling better."
 }
 [sha:ddcac14bb15624eff7ce3fc16ead5f14ffe8f953]{
 -- Page 22, Offset 8241.
 -- さっきの悲しそうな顔はどこにいったのか、先輩はくすくすと笑っている。
-While I wonder where her sad face from earlier went, Senpai chuckles.
+Senpai chuckles softly while I try and work out where the sad face from earlier disappeared to.
```
Comment (Avuxo) [inline script/Ciel/Day 4/04_01_CIEL04_1.txt:259]: from earlier went.

File: script/Ciel/Day 4/04_01_CIEL04_1.txt [hunk 7]
```diff
@@ -177,87 +192,101 @@ It's not like I hate you or anything."
...
-"...... Ciel-senpai?"
+"Ciel-senpai...?"
 }
 [sha:6fd68861835f0d842397f3ab2204ce9e4b062d8c]{
 -- Page 23, Offset 8243.
 -- 「冗談ですよ。わたしだって遠野くんのお家に直接電話なんかしません。乾くんにお願いして、昨夜遠野くんは乾くんの家に泊まった、という電話をしてもらったんです」
-"I'm just kidding. Even I wouldn't directly call your family like that. I asked Inui-kun to call and tell them you were staying at his house."
+"I was just kidding.
+ Even I wouldn't contact your family directly like that.
+ I asked Inui-kun to call them and say you were spending the night at his house."
```
Comment (Avuxo) [inline script/Ciel/Day 4/04_01_CIEL04_1.txt:271]: Arihiko NTR

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 4/04_01_CIEL04_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:201]: me and who
IMG_20200101_193955_MP

File: script/Ciel/Day 4/04_01_CIEL04_1.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:259]: I feel like disappeared to is more idiomatic? Maybe it's en_GB

File: script/Ciel/Day 4/04_01_CIEL04_1.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:166]: I kinda like the inline style? But maybe I am prejudiced against colons

Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:166]: my problem is with the "that". If you want to keep the line colon-free, just drop it

File: script/Ciel/Day 4/04_01_CIEL04_1.txt (hunk 6 above)
Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_1.txt:259]: sounds pretty wrong to me so i'd say it's probably an atlantic divide

**Pull request merged**